pkg/application/application.go

```diff
@@ -19,23 +19,20 @@ func getDefaultAppName() string {
 }
 
 // Create a new application
-func Create(client *occlient.Client, applicationName string) error {
-	project := project.GetCurrent(client)
+func Create(client *occlient.Client, appName string) error {
+
+	exists, _ := Exists(client, appName)
```

Comment:
why are we ignoring the err here? Should we not exit the program if a error occurs?

Reply:
we just need to check whether app exist or not, we don't care about error here